config/config_test.go

```diff
@@ -1611,13 +1611,13 @@ func TestConvertAs2_0(t *testing.T) {
 	}
 
 	for i, test := range tests {
-		cfg, r := ConvertAs2_0(test.in.cfg, "", nil)
+		cfg, r := Convert(test.in.cfg, "", nil)
 		assert.Equal(t, test.out.r, r, "#%d: bad report", i)
 		assert.Equal(t, test.out.cfg, cfg, "#%d: bad config", i)
 	}
 }
 
-func TestAndConvertAs2_0_0(t *testing.T) {
+func TestAndConvert_0(t *testing.T) {
```

Comment:
I thinkk this is supposed to be TestParseAndConvert

Reply:
oh yup, :s/ messed that one up.

Reply:
should I add parse there? it wasn't there before. I'm fine adding it if that makes sense.

Reply:
Yeah. I added that test; it's supposed to be TestParseAndConvert. Looks like I just typo'd it.